CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added jackson dependency to server ([#5366] (https://github.com/opensearch-project/OpenSearch/pull/5366))
 - Added experimental extensions to main ([#5347](https://github.com/opensearch-project/OpenSearch/pull/5347))
 - Adding support to register settings dynamically ([#5495](https://github.com/opensearch-project/OpenSearch/pull/5495))
+- Add minor bug fix for cluster manager throttling settings ([#5524](https://github.com/opensearch-project/OpenSearch/pull/5524))
```

Comment:
1. Please describe what this PR fixes. If you need more than one line (this PR fixes 2 things), it's ok to mention the GH pull request twice.
2. Move this to the Fixes section.
3. Is it going to be backported to 2.x? Looks backwards compatible but I could be wrong. If so, please move it to the 2.x bection below.

Reply:
Ack. Will make this changes.